hbase_storage/hbase1.1.8/src/main/java/com/splicemachine/storage/Hbase10PartitionCache.java

```diff
@@ -63,5 +64,27 @@ public void configure(SConfiguration configuration){
     public void put(TableName tableName, List<Partition> partitions) throws IOException {
         partitionCache.put(tableName,partitions);
     }
+
+    @Override
+    public void invalidateAdapter(TableName tableName) throws IOException {
+        partitionAdapterCache.invalidate(tableName);
+        ((ClusterConnection)HBaseConnectionFactory.getInstance(config).getConnection()).clearRegionCache(tableName);
```

Comment:
Need to change the use of HConnection in the function invalidate() to ClusterConnection to be consistent with invalidateAdapter. We got into compilation error due to this for the special branch-2.5-1820c.

Reply:
@yxia92 I'm away from my computer, could you (or somebody else) fix it, if merging this is urgent? Thanks

Reply:
I can push a fix for the compilation issue.